uw-frame-components/config.js

```diff
@@ -1,51 +1,71 @@
 'use strict';
-define([], function() {
-  return {
+define(['./my-app/app-config'], function(myAppConfig) {
```

Comment:
For reasons I don't fully grok, I think you need to add the ".js" suffix in situations like this where the runtime is not fully bootstrapped.